llvm/test/CodeGen/X86/GlobalISel/isel-fcmp-i686.mir

```diff
@@ -0,0 +1,2784 @@
+# NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+# NOTE: This MIR test is required because the support for 64 bit memory ops is missing in i686 mode, Due to distinction between float/int types, support is expected in near future and there is this RFC in place https://discourse.llvm.org/t/rfc-globalisel-adding-fp-type-information-to-llt/83349. Once this support is introduced this test must be dropped and integrated into the LLVM IR tests.
+# RUN: llc -O2 -mtriple=i686-linux-gnu -mattr=+x87,-sse,-sse2 -start-after=legalizer -disable-gisel-legality-check -global-isel -verify-machineinstrs %s -o - | FileCheck %s --check-prefixes GISEL-X86
+
+--- |
+  ; ModuleID = 'c2.ll'
```

Comment:
Let's simplify .mir test.
1) I suppose there is no need to test float if we have problem only with 64 bit mem ops.
2) Let's drop IR
3) Instead of checking final asm, let's check the first MIR after selection. We usually place MIR checks before MIR input per function, please take a look at other .mir tests nearby
4) Let's drop unnecessary attributes, registers, etc. e.g., `hasEHScopes`